frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -1,6 +1,5 @@
 .LemonButton {
     color: var(--primary);
-    font-weight: 600;
```

Comment:
This should still be the case!

Reply:
Hm, just curious what is the reason we enforce this on all buttons? It seems like something that should be off by default since a lot of the buttons in the Figma mockups have normal font weights.